lib/svd_quda.h

```diff
@@ -291,6 +291,9 @@ void getRealBidiagMatrix(const Matrix<Cmplx,3> & mat,
   typename RealTypeId<Cmplx>::Type beta;
   Cmplx w, tau, z;
 
+  u(0,0).x = -2.0;
```

Comment:
using -2.0 to detect special cases should come with a comment explain what is done